Documentation/gettingstarted/index.rst

```diff
@@ -21,6 +21,7 @@ complete will help you to get started in your area of choice:
    :glob:
 
    minikube
+   dns 
```

Comment:
is this needed? Ideally we keep the uppercase "DNS" but don't allow the lowercase "dns".